public/static/docs/command-reference/list.md

```diff
@@ -0,0 +1,138 @@
+# list
+
+List <abbr>repository</abbr> contents, including files and directories tracked
+by DVC (<abbr>data artifacts</abbr>) and by Git.
+
+## Synopsis
+
+```usage
+usage: dvc list [-h] [-q | -v] [-R] [--outs-only] [--rev [REV]] url [path]
+
+positional arguments:
+  url              Location of DVC repository to list.
+  path             Path to a file or directory within the repository.
```

Comment:
Quick question: is it `path` or `target` @JIoJIaJIu? And if it's `target`, why was that decided, instead of `path` like in `dvc import/get`? Thanks

Reply:
Or maybe just `dir` not to overload any of the existing param names too much.

Related: iterative/dvc/issues/3422

Reply:
> And if it's target

It is `path`, I'm waiting for the merge of https://github.com/iterative/dvc.org/pull/1021 for applying naming changes in terms of the https://github.com/iterative/dvc/issues/3381

Reply:
I see, based on my own review which I forgot about haha: https://github.com/iterative/dvc/pull/3246#discussion_r382870261

> I'm waiting for the merge of #1021 for applying naming changes

Have you considered opening a PR from a new branch into dvc_list while waiting for approvals? 😬 That way we could compare the changes here on that PR as well. Up to you though but otherwise please come back to this docs later PR to apply the further `list` updates, if any.

Reply:
Wait, I think I didn't understand you correctly. You're waiting for this docs PR to apply changes to core? That's probably not necessary, please go ahead with your core changes, and we'll compare agains this one accordingly. No need to block your work I think.

Reply:
> You're waiting for this docs PR to apply changes to core

Nope, I was waiting https://github.com/iterative/dvc/pull/3351 cause it can be difficult to merge changes. Now it's merged and I am going to apply changes